<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>General 5ho</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/dron.jpeg</normaloff>../icon/dron.jpeg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="WS_scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>100</width>
      <height>310</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>308</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="Snapshot_scrollArea">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>60</y>
      <width>142</width>
      <height>310</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>140</width>
       <height>308</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Menu1_pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>menu1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Menu2_pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>menu2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Menu3_pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>menu3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Menu4_pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>menu4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Menu5_pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>menu5</string>
    </property>
   </widget>
   <widget class="QLabel" name="Preview_Label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>310</width>
      <height>310</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../icon/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="Top_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(147, 147, 147);
background-color: rgb(184, 184, 184);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="Right_frame">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>160</width>
      <height>330</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollArea" name="Active_scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>130</width>
       <height>270</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>128</width>
        <height>268</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="Active_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>295</y>
       <width>130</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(51, 160, 255);</string>
     </property>
     <property name="text">
      <string>Active</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>60</y>
      <width>20</width>
      <height>310</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>Right_frame</zorder>
   <zorder>Top_frame</zorder>
   <zorder>WS_scrollArea</zorder>
   <zorder>Snapshot_scrollArea</zorder>
   <zorder>Menu1_pushButton</zorder>
   <zorder>Menu2_pushButton</zorder>
   <zorder>Menu3_pushButton</zorder>
   <zorder>Menu4_pushButton</zorder>
   <zorder>Menu5_pushButton</zorder>
   <zorder>Preview_Label</zorder>
   <zorder>line</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
